examples/stable_diffusion_xl/train.py

```diff
@@ -194,6 +200,7 @@ def train(args):
         raise ValueError("args.ms_mode value must in [0, 1]")
 
     # 5. Start Training
+    assert args.max_num_ckpt is None or args.max_num_ckpt > 0, "args.max_num_ckpt must be None or a positive integer!"
```

Comment:
for best practise, using raise Exception is better than asset

Reply:
Updated